satpy/etc/composites/li.yaml

```diff
@@ -1,5 +1,8 @@
 ---
-sensor_name: visir/li
+# we use li only here, and not visir/li, since the second can cause dependency issues when creating composites
+# combined with imagers in a multi-reader Scene. visir composites do not apply to LI anyway.
+sensor_name: li
```

Comment:
We could maybe define (I don't know if it errors) a new type of instrument. Like `li/li` for "lightning imager" type and the instrument is called "lightning imager". Or `lightning/li`? Not a huge deal for now but something to think about.

Reply:
True, yes. For now we don't have any, but it would be indeed nice to design some shared composites that work also with GLM and/or other networks.